tests/runtests.py

```diff
@@ -32,9 +34,20 @@
 LOCAL_BASES = [
     'https://w3c.github.io/json-ld-api/tests',
     'https://w3c.github.io/json-ld-framing/tests',
-    'https://github.com/json-ld/normalization/tests'
+    'https://w3c.github.io/rdf-canon/tests/',
 ]
 
+
+def default_test_targets() -> List[Path]:
```

Comment:
How about changing this so it works something like the [jsonld.js test runner](https://github.com/digitalbazaar/jsonld.js/blob/main/tests/test-node.js#L25) and checks for the local test suite dir, then for the sibling dir if not found.  That allows for alternate workflows.

Reply:
It didn't consider missing test suites an error before.  I'm not sure what should happen for that.  Never seemed a problem in practice.

Reply:
Do you believe there is a use case for it? Probably the main goal of turning specifications into submodules is to automate the connection to them, standardizing it across the developers' clones of `pyld` repo.

Having the specifications cloned into the root directory of the repo, for instance, will require `.gitignore` rules (or it's way too easy to commit them, which is likely not what one wants).

We can later add an error message to this function. If the default test locations are used, and if tests are missing, then probably something is wrong. For instance, the developer forgot to check out the submodules. Their tests will be incorrectly green locally, and then, after a push, failing CI will pose an unpleasant surprise.